examples/java-matter-controller/BUILD.gn

```diff
@@ -21,23 +21,17 @@ import("${chip_root}/build/chip/tools.gni")
 
 android_binary("java-matter-controller") {
   output_name = "java-matter-controller"
-
   deps = [
-    ":android",
+    "${chip_root}/examples/java-matter-controller/stub_src",
```

Comment:
stub_src is used by src/controller/java, not the java-matter-controller example app, I think we should move it to src/controller/java/src/stub so that this component could be used by other Java application

Reply:
we would not expect any other java test application.